snuba/datasets/processors/errors_processor.py

```diff
@@ -133,7 +134,16 @@ def process_insert(
             or {"values": []}
         )
         stacks = exception.get("values", None) or []
-        self.extract_stacktraces(processed, stacks)
+
+        thread: SentryThreadChain = data.get(
+            "threads",
+            cast(
+                SentryThreadChain, data.get("sentry.interfaces.threads.Threads", None)
+            ),
+        ) or {"values": []}
+        threadsStack = thread.get("values", None) or []
+
+        self.extract_stacktraces(processed, stacks, threadsStack)
```

Comment:
@evanh I have no idea if this is right, I just kinda copied based on the `exception` example.
Does `sentry.interfaces.threads.Threads` even exist? I have not found the usage anywhere else.

Reply:
Where is the definition of `sentry_kafka_schemas.schema_types.events_v1`?

Reply:
That module is code-generated from https://github.com/getsentry/sentry-kafka-schemas/blob/main/schemas/events.v1.schema.json -- we can pair tomorrow on fixing it up to contain threads. You should be able to add a minimal type that covers the fields you need.

Reply:
> Does sentry.interfaces.threads.Threads even exist? I have not found the usage anywhere else.

no, this is dead code. you can remove it. everything with `sentry.interfaces` is legacy

Reply:
Alright, let's pair up when you can.
Started something here https://github.com/getsentry/sentry-kafka-schemas/pull/45/files